docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.9'
+
+services:
```

Comment:
We need to mention the new Dockerfile and compose methods in the relevant readme sections